addons/mail/static/tests/activity/activity.test.js

```diff
@@ -404,6 +404,41 @@ test("activity click on edit", async () => {
     await assertSteps(["do_action"]);
 });
 
+QUnit.test("activity click on edit should pass correct context", async (assert) => {
```

Comment:
This becomes `test()` in 17.2 rather than `QUnit.test()`.

This is because we used QUnit but in 17.2 tests are written with HOOT.